package.json

```diff
@@ -36,53 +38,61 @@
   "license": "ISC",
   "engines": {
     "node": ">= 6",
```

Comment:
Shouldn't this be node 7? Node 6 doesn't come with npm >= 4 built-in.

Reply:
This or, simply:

```
"node": ">=8"
"npm": ">=5"
```

Which is much faster, comes with built-in package-lock.json, etc.